frontend/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "proxy": "http://nanuda.herokuapp.com",
```

Comment:
@geooff I was running into CORS issues and had to do this to bypass it locally. Is there a way you can enable CORS on your end?

Reply:
I think so, let me look into this a bit. Is CORS stuff blocking this PR?